packages/docs/src/repl/repl-share-url.ts

```diff
@@ -59,11 +67,11 @@ export const createPlaygroundShareUrl = (data: PlaygroundShareUrl) => {
   params.set('entryStrategy', data.entryStrategy);
 
   const filesStr = JSON.stringify(data.files);
-  // const filesBuf = strToU8(filesStr);
-  // const compressedUint8Array = compressSync(filesBuf);
-  // const compressedString = strFromU8(compressedUint8Array);
-  const encodedURI = encodeURIComponent(filesStr);
-  const filesBase64 = btoa(encodedURI);
+  const filesBuf = strToU8(filesStr);
+  const compressedUint8Array = compressSync(filesBuf);
+  const compressedString = strFromU8(compressedUint8Array, true);
```

Comment:
Since the resulting string will contain only characters from the latin-1 set, there's no need to run `encodeURIComponent` with this result.

Running `encodeURIComponent` here will actually produce larger URLs.